queries.sql

```diff
@@ -108,3 +108,62 @@ select species, avg(escape_attempts) from animals where extract(year from  date_
  species |        avg         
 ---------+--------------------
  Pokemon | 3.0000000000000000
+
+-- What animals belong to Melody Pond?
+SELECT name FROM animals inner join owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT animals.name FROM animals join species ON species.id = animals.species_id WHERE species.name='Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT owners.full_name AS owner, animals.name FROM animals right join owners ON owners.id = animals.owner_id;
+
+-- How many animals are there per species?
+SELECT species.name AS specie, count(animals.name) FROM animals inner join species ON species.id= animals.species_id group by species.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT owners.full_name AS owner, animals.name AS animal FROM animals inner join owners ON owners.id= animals.owner_id 
+        inner join species ON species.id=animals.species_id WHERE owners.full_name='Jennifer Orwell' and species.name='Digimon';
+
+-- List all animals owned by Dean Winchester that haven't tried to escape.
+SELECT animals.name FROM animals inner join owners ON owners.id=animals.owner_id WHERE owners.full_
+
+-- Who owns the most animals? --Melody Pond
+
+SELECT owners.full_name AS owner, COUNT(*) AS animal_QTY FROM animals inner join owners ON owners.id=animals.owner_id group BY owners.full_name;
+
+-- Who was the last animal seen by William Tatcher?
+SELECT animals.*, visits.date_of_visit FROM animals LEFT JOIN visits ON animals.id = visits.animals_id LEFT JOIN vets ON visits.vets_id =vets.id WHERE vets.id = 
+(SELECT id FROM vets WHERE vets.name = 'William Tatcher') ORDER BY date_of_visit DESC;
+
+-- How many different animals did Stephanie Mendez see
+SELECT vets.name, COUNT(animals.*) FROM animals LEFT JOIN visits ON animals.id = visits.animals_id LEFT JOIN vets ON visits.vets_id = vets.id WHERE vets.id = 
+(SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez') GROUP BY vets.name;
+
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT vets.name, species.name FROM vets FULL JOIN specializations ON vets.id = specializations.vets_id LEFT JOIN species ON specializations.species_id = species.id;
+
+-- List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
+SELECT animals.*, visits.date_of_visit FROM animals LEFT JOIN visits ON animals.id = visits.animals_id LEFT JOIN vets ON visits.vets_id = vets.id WHERE vets.id =
+ (SELECT id FROM vets WHERE vets.name = 'Stephanie Mendez') AND visits.date_of_visit BETWEEN '04-01-2020' AND '08-30-2020' ORDER BY date_of_visit DESC;
+
+
+-- What animal has the most visits to vets?
+SELECT animals.name, COUNT (animals.id) FROM animals LEFT JOIN visits ON animals.id = visits.animals_id GROUP BY animals.id ORDER BY COUNT (animals.id) DESC;
+
```

Comment:
- [ ] This query should return the animal with the most visits to vets. Your query will show all animals according to the frequency of visits to vets, but you should show one animal. I would advise you to use `LIMIT 1` at the end of the query. 👍